requirements.yml

```diff
@@ -0,0 +1,5 @@
+- src: geerlingguy.filebeat
+- src: geerlingguy.java
+- src: https://github.com/jesusaurus/ansible-role-logstash
+  version: arbitrary-outputs
+  name: geerlingguy.logstash
```

Comment:
Yeah, I'm on the fence about using these. It might be better to commit to forking and stripping out what we don't want/need. And we definitely need a larger discussion.